packages/cli/src/commands/exec.js

```diff
@@ -12,6 +12,14 @@ import { generatePrismaClient } from 'src/lib/generatePrismaClient'
 const runScript = async (scriptPath, scriptArgs) => {
   const script = await import(scriptPath)
   await script.default({ args: scriptArgs })
+
+  try {
+    const { db } = await import(path.join(getPaths().api.lib, 'db'))
+    db.$disconnect()
+  } catch (e) {
+    // silence
+  }
+
```

Comment:
Super weird, we should definitely add a comment here to explain what we're doing.